driver/main_args.ml

```diff
@@ -368,6 +368,14 @@ let mk_warn_help f =
   "-warn-help", Arg.Unit f, " Show description of warning numbers"
 ;;
 
+let mk_color f =
+  "-color", Arg.Unit f,
+  " <bool> Enable ANSI color codes for warning/error messages"
+
+let mk_no_color f =
+  "-no-color", Arg.Unit f,
+  " <bool> Disable ANSI color codes for warning/error messages"
+
```

Comment:
It would be nice if there was a way to explicitly ask on the command line the "auto" mode. You could have `-color-auto` as a third flag, or maybe accept `-color {on,off,auto}`. This would mean that the `OCAMLPARAM` handling code would have to be changed again...